bot/resources/tags/under.md

```diff
@@ -0,0 +1,35 @@
+**Use cases of Underscore in Python**
+
+- **Dunders**: Double Underscore (Dunder) methods are special methods defined in a class that are invoked implicitly. For example, `__name__`,`__init__`,`__repr__`, and `__str__`.
+Use the `!dunder-methods` tag to know more about these methods
+- **Single leading Underscore**:  In front of a variable, a function, or a method name means that these objects are used internally, such as `_name`. Remember that these objects can be accessed outside or other script.
+- **Single leading Underscore**:  In front of a variable, a function, or a method name means that these objects are used internally, such as `_name`. However, remember that these objects **can** be accessed outside.
+-  **Double leading underscores**: Double leading underscores are typically used for name mangling.
+Name mangling is a process by which the interpreter changes the attribute name to avoid naming collisions in subclasses.[GeeksforGeeks](https://www.geeksforgeeks.org/name-mangling-in-python/)
```

Comment:
I think  you should say something like: More details & examples about name mangling can be found [here](https://www.geeksforgeeks.org/name-mangling-in-python/).